SConstruct

```diff
@@ -586,25 +609,138 @@ def list_sources(base, suffixes):
                     yield os.path.normpath(path)
     return list(_iter(base))
 
-def add_soci_sources(env, object_builder, is_unity):
+def append_sources (result, *filenames, **kwds):
+    result.append([filenames, kwds])
+
+def get_soci_sources(style):
+    result = []
     cpp_path = [
         'src/soci/src/core',
         'src/sqlite',]
-    object_builder.add_source_files(
+    append_sources(result,
         'src/ripple/unity/soci.cpp',
         CPPPATH = cpp_path)
-    if is_unity:
-        object_builder.add_source_files(
+    if 'unity' == style:
+        append_sources(result,
             'src/ripple/unity/soci_ripple.cpp',
             CPPPATH = cpp_path)
+    return result
+
+def get_classic_sources():
+    result = []
+    append_sources(result,
+        *list_sources('src/ripple/app', '.cpp'),
+        CPPPATH=[
+             'src/soci/src/core',
+             'src/sqlite',]
+    )
+    append_sources(result,
```

Comment:
I don't think this is PEP 8... https://www.python.org/dev/peps/pep-0008/#indentation

You have several choices, one of which is to move `result,` to join the rest of the arguments.